apps/codeforafrica/src/payload/collections/Pages.js

```diff
@@ -0,0 +1,41 @@
+import fullTitle from "../fields/fullTitle";
+import slug from "../fields/slug";
+import formatDraftUrl from "../utils/formatDraftUrl";
+
+import { pages } from "./slugNames";
+
+const Pages = {
+  slug: pages,
```

Comment:
Not sure this is necessary. We can just set it to "pages". No need to declare slug in a different file